contrib/fullnode_config.ini

```diff
@@ -2,11 +2,15 @@
 log-console-appender = {"appender":"stderr","stream":"std_error"}
 
 # File appender definition json:  {"appender", "file"}
-log-file-appender = {"appender":"p2p","file":"logs/p2p/p2p.log"}
+log-file-appender = {"appender":"sophiatxd","file":"logs/sophiatxd.log"}
 
 # Logger definition json: {"name", "level", "appender"}
-log-logger = {"name":"default","level":"warn","appender":"stderr"}
-log-logger = {"name":"p2p","level":"warn","appender":"p2p"}
+log-logger = {"name":"default","level":"info","appender":"stderr"}
```

Comment:
How quickly does the fullnode logs grow with loglevel info?

Reply:
should be alright, but i will check it when it is running on devnet...